### ISSUE TITLE
Accessibility Tests Failing

### ISSUE THREAD

**Issue by weiquu** [TEAMMATES/teammates#12644]
Accessibility tests are failing due to the `rubric-question-response` component. In particular, the body of the table has id `rubric-answers`, but this component appears for each rubric question response, meaning that there might be multiple `rubric-answers` ids on the same page.

To re-create such a page, create a feedback session with rubric questions, respond to the session with a couple of students, then go to student pages and view responses for the feedback session.

Any fix should note that the `FeedbackResultsPage` E2E test makes use of the `rubric-answers` id, so there needs to be some changes there as well.
